src/body/to_bytes.rs

```diff
@@ -13,6 +13,28 @@ use super::HttpBody;
 /// Care needs to be taken if the remote is untrusted. The function doesn't implement any length
 /// checks and an malicious peer might make it consume arbitrary amounts of memory. Checking the
 /// `Content-Length` is a possibility, but it is not strictly mandated to be present.
+///
+/// Create an empty `Body` stream.
+///
+/// # Example
+///
+/// ```
+/// let response = client.request(req).await?;
```

Comment:
You'll need to put a couple "hidden" lines above to allow the usage of `.await` and `?`. Something like:

```
/// # async fn ex() -> Result<(), Box<dyn std::error::Error>> {
```

And then close the function in another hidden line at the bottom.